managing_providers/index.md

```diff
@@ -8,7 +8,7 @@
 In {{ site.data.product.title_short }}, a *provider* is an external virtualization, cloud, or containers environment that manages multiple virtual machines
 or instances residing on multiple hosts. One example is Red Hat Virtualization, a platform that manages multiple hosts and virtual machines.
 
-In {{ site.data.product.title_short }}, a *manager* is an external management environment that manages more than one type of resource. One example of a manager is OpenStack, which manages infrastucture, cloud, network, and storage resources.
+In {{ site.data.product.title_short }}, a *manager* is an external management environment that manages more than one type of resource. One example of a manager is OpenStack, which manages infrastructure, cloud, network, and storage resources.
```

Comment:
Found this typo when my editor was suggesting "infrastucture" as an autocomplete option